client/src/assets/js/app/typing-lesson.js

```diff
@@ -136,6 +154,7 @@
       function startWpmTimer() {
         wpmTimer = $interval(function() {
           if (scope.lesson.state === LevelStates.Post) {
+            scope.lesson.incorrectCharsSorted();
```

Comment:
I think we should do this _after_ stopping the timer (on the line below).  It's a higher priority and is cheap efficiency-wise.